homeassistant/components/intelliclima/const.py

```diff
@@ -0,0 +1,33 @@
+"""Constants for the IntelliClima integration."""
+
+import logging
+
+LOGGER = logging.getLogger(__package__)
+
+DOMAIN = "intelliclima"
+
+# Update interval
+DEFAULT_SCAN_INTERVAL = 10  # seconds
```

Comment:
10 seconds seems very short to me considering we are using a remote API. What do you think about 1 minute? This value is recommended by handbook for most user cases: 
https://developers.home-assistant.io/docs/core/integration-quality-scale/rules/appropriate-polling